src/Entities/TestClasses/Hand.test.ts

```diff
@@ -1,15 +1,65 @@
-import BellePlaineRulesCardRanker from '~Entities/BellePlaineRulesCardRanker'
+import BellePlaineRulesCardRanker from '../BellePlaineRulesCardRanker'
 import Card from '../Card'
 import Hand from '../Hand'
 
 describe('Hand', () => {
```

Comment:
A few notes on the testing files.

`describe` is just a way of grouping the tests, as in "describe what is being tested"
`it` refers to a single test and I write them as `it('Should do x')` basically a sentence explaining what functionality is demonstrated by the body of the test.

`beforeEach` runs before each test as a way of cleaning up so that the tests don't influence each other.